src/app/auth.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { AngularFireAuth } from '@angular/fire/auth';
+import { FirebaseService } from './firebase.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  userCredential: Promise<void | firebase.auth.UserCredential>;
+
+  constructor(public auth: AngularFireAuth, public firebaseService: FirebaseService) { }
```

Comment:
These shouldn't be public
```suggestion
  constructor(private auth: AngularFireAuth, private firebaseService: FirebaseService) { }
```